src/store/quoteReplyStore.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+ *
+ * @author Marco Ambrosini <marcoambrosini@pm.me>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import Vue from 'vue'
+
+const state = {
+	messagesToBeReplied: {}
```

Comment:
What is this store used for?
Why do you store quotes separately from usual messages?

Reply:
1) So the way I did it is I have a store of messages to be replied to on a per-conversation basis. The very fact that there's a { 'token' : messagetobereplied } key value pair in that store means that there's an undergoing quote reply operation. Also this allows not to end the quote-reply operation when you exit and re-enter the conversation.
2) To keep files shorter

Reply:
I think the names are really confusing, but the logic make sense!
Nice feature to be able to keep unsent messages! :+1:

Reply:
Yeah sorry I always struggle with names.. Do you have any suggestion in mind?

Reply:
PendingMessagesStore.js? :woman_shrugging:

Reply:
That's going to be a different one. This is just pending messages to be replied :)

Reply:
Be careful about keeping things simple though! :)
If even I get confuse about this, I'm guessing newcomers won't have it easier ;)